homeassistant/components/elmax/config_flow.py

```diff
@@ -40,6 +63,43 @@
     }
 )
 
+CHOOSE_MODE_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_ELMAX_MODE, default=CONF_ELMAX_MODE_CLOUD): SelectSelector(
+            SelectSelectorConfig(
+                options=[
+                    SelectOptionDict(
+                        value=CONF_ELMAX_MODE_CLOUD,
+                        label="Connect to Elmax Panel via Elmax Cloud APIs",
+                    ),
+                    SelectOptionDict(
+                        value=CONF_ELMAX_MODE_DIRECT,
+                        label="Connect to Elmax Panel via local/direct IP",
+                    ),
+                ],
+                mode=SelectSelectorMode.DROPDOWN,
+            )
+        )
+    }
+)
```

Comment:
I think a radio button would work well here. You could also make the labels translatable